assets/cluster-monitoring-operator/monitoring-targets-edit-cluster-role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: monitoring-targets-edit
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  - podmonitors
```

Comment:
maybe I'm missing context, but I would have expected a role for monitoring configurations, that is not just about targets but both scraping and alerting configuration

Reply:
For configuration we specified a dedicated role, see https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/user-workload-monitoring.md#monitoring-targets-edit, and https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/user-workload-monitoring.md#monitoring-workload-config-edit:

`monitoring-targets-edit`: This is a cluster role which can be bound against a concrete namespace by the cluster admin. Embeds the get/create/edit/delete permissions for the following custom resources:

- ServiceMonitor
- PodMonitor
- PrometheusRule

It allows to create new scraping targets for services/pods and allows to create new recording or alerting rules.

`monitoring-workload-config-edit`:
This is a named role against the workload-monitoring-config configmap resource in the openshift-monitoring namespace embedding get/create/edit/delete permissions.

Reply:
Or did we miss something else here? :thinking: